### DISCUSSION TITLE
How does faster-whisper compare to DeepSpeech?

### DISCUSSION THREAD

**Discussion by Geremia** [SYSTRAN/faster-whisper#401] [General]
How does faster-whisper compare to [DeepSpeech](https://deepspeech.readthedocs.io)?
Are there benchmarks on this?

**Comment by silvacarl2** [comment]
It blows it away in terms of price.  8-)

**Reply by etlweather** [reply]
Free versus free?

**Comment by rjwilmsi** [comment]
I tried DeepSpeech before whisper (or faster-whisper) was released.

DeepSpeech in my testing had substantially worse accuracy compared to original whisper. Below is a comparison I did of the output of deepspeech versus original whisper small model, for youtube video https://www.youtube.com/watch?v=GFu64hnqzVo with text converted to lowercase and punctuation removed to simplify comparison. DeepSpeech makes many mistakes, original whisper small makes about 2 small mistakes in 1000 words. I have not done the same comparison with faster-whisper.

Maybe DeepSpeech has improved since then, I haven't retested.

![Deepspeech v whisper](https://github.com/guillaumekln/faster-whisper/assets/115937415/f0ccf543-ad73-4f9b-a191-9f559c6f6131)
